frontends/src/desktop/components/views/CredentialList.svelte

```diff
@@ -51,13 +51,13 @@
 
 <div class="grow px-16 py-4">
 	<div class="h-full flex flex-wrap content-start gap-3">
-		{#each updatedCredentials as credential}
+		{#each updatedCredentials as credential (credential.id)}
 			{@const credentialType = credential.data.credentialType}
 			{@const categoryInfo = CATEGORIES.find(
 				(item) => item.type === credentialType,
```

Comment:
would it help to make categories as a json config with type as the key?

Reply:
Lest consider that in future?